analysis/estimate/notebooks/run-estimate.Rmd

```diff
@@ -0,0 +1,317 @@
+
+### Compare measures from pseudobulk and bulk
+
+Since bulk scores were very similar, we'll proceed to visualize only the bulk counts.
+
+```{r}
+estimate_sub_df <- estimate_df |>
+  dplyr::filter(expression_type != "bulk_tpm")
+```
+
+First, we'll look at both the distribution of scores, and then the relationship between pseudobulk and bulk scores.
+
+#### Score distributions
+
+```{r fig.height=4, fig.width=12}
+ggplot(estimate_sub_df) + 
+  aes(x = score_type, y = score, color = expression_type) + 
+  geom_jitter(
+    position = position_jitterdodge(jitter.height = 0, jitter.width = 0.1), 
+    size = 0.75
+  ) +
+  scale_color_brewer(palette = "Set2") +
+  facet_wrap(vars(project_id), scales = "free_x", nrow = 1) + 
+  theme(legend.position = "bottom") + 
+  guides(color = guide_legend(override.aes = list(size = 2)))
+```
+
+First, what do we see for immune scores?
+
+* For SCPCP000001 and SCPCP000002, pseudobulk scores tend to be higher than bulk scores
+  * However, for SCPCP000001, we have a sort of trimodal distribution of values - it would be nice if this corresponded to diagnoses?!
+* For the other three projects, pseudobulk scores appear to be somewhat lower than bulk
+
+Second, what do we see for stromal scores?
+
+* For SCPCP000001 and SCPCP000002, pseudobulk scores look either on par with bulk or _maybe_ a bit lower, but that is mostly in SCPCP000001 and one outlier in SCPCP000002
+* For the other three projects, pseudobulk and bulk scores appear highly similar
+
+
+#### Compare modalities
+
+```{r fig.width=10}
+compare_modalities_df <- estimate_df |>
+  tidyr::pivot_wider(
+    names_from = expression_type, 
+    values_from = score
+  ) 
+
+ggplot(compare_modalities_df) + 
+  aes(x = bulk_counts, y = pseudobulk) + 
+  geom_point() +
+  geom_abline(color = "darkorange2") +
+  geom_smooth(method = "lm") +
+  facet_grid(
+    rows = vars(score_type), 
+    cols = vars(project_id)
+  ) +
+  labs(title ="pseudobulk ~ bulk counts") +
+  theme(aspect.ratio = 1)
+```
+
+* There does not appear to be a significant relationship between modalities for SCPCP000002 and SCPCP000006, for either score 
+  * That said, it is worth noting that all samples' immune scores, and most samples' stromal scores, for SCPCP000002 are higher in pseudobulk than in bulk
+* For SCPCP000001, immune scores tend to be higher in pseudobulk but stromal scores are about the same between modalities
+* For SCPCP000006 and SCPCP0000017, immune scores tend to be lower in pseudobulk but stromal scores are about the same between modalities
+* Note that projects SCPCP000001 and SCPCP000002 are single-cell, but the other three projects are single-nucleus
+
+#### Compare across sequencing unit
+
+On the single-cell level, some projects used single-cell libraries and others used single-nucleus libraries. 
```

Comment:
This doesn't surprise me one bit and it's a nice validation to see that single-cell you get more immune cells than single-nuclei!